### DISCUSSION TITLE
How to put a Python wrapper around Slick-Grid universal example11 ?

### DISCUSSION THREAD

**Discussion by richlysakowski** [ghiscoding/slickgrid-universal#964] [General]
I really like the full functionality of Slick-grid universal.   

Does anyone know how to put Python wrappers around these examples, so that it could be used in JupyterLab or JupyterNB, or VSCode? 

Suggestions for how to get started?  

Thank you. 

Rich

**Comment by ghiscoding** [comment]
Sorry I am not a Python user and I have no clue, you should ask on Stack Overflow. I have seen some questions and answers with SlickGrid and Python

**Reply by richlysakowski** [reply]
One of the founders of Streamlit has wrapped SlickGrid inside of a Streamlit Component.  Perhaps that project could be used to extract a working Python  standalone wrapper for SlickGrid.  

I will put the link to the Streamlit project here soon.

**Reply by ghiscoding** [reply]
I know and heard about Streamlit-Slickgrid and I've seen their youtube video demoing it:
https://www.youtube.com/watch?v=fsJstnDU-5U

the sad thing though, is that after creating the project, the maintainers haven't put too much work in it afterward :(

and here's the Streamlit-Slickgrid repo
https://github.com/streamlit/streamlit-slickgrid
